src/SprykerEco/Zed/PunchoutCatalogs/Communication/Controller/EntryPointsController.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\PunchoutCatalogs\Communication\Controller;
+
+use Generated\Shared\Transfer\PunchoutCatalogEntryPointFilterTransfer;
+use Spryker\Zed\Kernel\Communication\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * @method \SprykerEco\Zed\PunchoutCatalogs\Persistence\PunchoutCatalogsRepositoryInterface getRepository()
+ * @method \SprykerEco\Zed\PunchoutCatalogs\Business\PunchoutCatalogsFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\PunchoutCatalogs\Communication\PunchoutCatalogsCommunicationFactory getFactory()
+ */
+class EntryPointsController extends AbstractController
+{
+    /**
+     * @uses \SprykerEco\Zed\PunchoutCatalogs\Communication\Controller\IndexController::indexAction()
```

Comment:
Let's keep `@uses` for cases when one constant uses the value of another, and `@see` for cases, when you want to mention the place, where this constant is used.

However, it looks really strange, that you add such annotation to protected constantly.

I assume:
    a) annotation is not needed here
or
    b) constant should be public

Reply:
It for sure needs to be protected
Replaced by @see